examples/timeline/other/verticalScroll.html

```diff
@@ -0,0 +1,118 @@
+<html>
+<head>
+  <title>Timeline | Vertical Scroll Option</title>
+
+  <script src="../../../dist/vis.js"></script>
+  <link href="../../../dist/vis.css" rel="stylesheet" type="text/css" />
+
+
+  <style type="text/css">
```

Comment:
I'm not sure haw other examples handle this, but I think we don't need this style here. The reason is to keep the examples as clean an copyable as possible.